server-phoenix/lib/helios_web/test_data/traffic_cams.ex

```diff
@@ -1,5 +1,5 @@
+    "[{\"id\":1,\"title\":\"I-95 at Broad St\",\"url\":\"http://www.dot.ri.gov/img/travel/camimages/95-21%20I-95%20S%20@%20Broad%20St%20.jpg\",\"location_id\":1,\"created_at\":\"2022-03-08T15:39:35.850Z\",\"updated_at\":\"2022-03-08T15:39:35.850Z\",\"feed_format\":\"image\"},{\"id\":2,\"title\":\"I-95 at Lonsdale\",\"url\":\"http://www.dot.ri.gov/img/travel/camimages/95-26%20I-95%20N%20@%20Lonsdale%20Ave.jpg\",\"location_id\":1,\"created_at\":\"2022-03-08T15:39:35.852Z\",\"updated_at\":\"2022-03-08T15:39:35.852Z\",\"feed_format\":\"image\"},{\"id\":3,\"title\":\"I-95 at Kinsley\",\"url\":\"http://www.dot.ri.gov/img/travel/camimages/95-22b%20I-95%20N%20@%20Kinsley%20Ave.jpg\",\"location_id\":1,\"created_at\":\"2022-03-08T15:39:35.854Z\",\"updated_at\":\"2022-03-08T15:39:35.854Z\",\"feed_format\":\"image\"},{\"id\":4,\"title\":\"Exit 10 (near I-295)\",\"url\":\"http://www.dot.ri.gov/img/travel/camimages/95-11%20I-95%20N%20@%20Toll%20Gate%20Rd.jpg\",\"location_id\":1,\"created_at\":\"2022-03-08T15:39:35.865Z\",\"updated_at\":\"2022-03-08T15:39:35.865Z\",\"feed_format\":\"image\"},{\"id\":5,\"title\":\"I-95 I-195 Split\",\"url\":\"http://www.dot.ri.gov/img/travel/camimages/195-1%20I-195%20W%20Split%20@%20I-95.jpg\",\"location_id\":1,\"created_at\":\"2022-03-08T15:39:35.867Z\",\"updated_at\":\"2022-03-08T15:39:35.867Z\",\"feed_format\":\"image\"},{\"id\":6,\"title\":\"Dean Street\",\"url\":\"https://www.dot.ri.gov/img/travel/camimages/6-1%20Rt%206%20E%20%20%2010%20N%20@%20Dean%20St.jpg\",\"location_id\":1,\"created_at\":\"2022-03-08T15:39:35.868Z\",\"updated_at\":\"2022-03-08T15:39:35.868Z\",\"feed_format\":\"image\"}]"
```

Comment:
Looks like we aren't storing these links in the database for the elixir version. Probably for the best, this seeding pattern seemed more complicated for our use case. Is there a way to make this blob of text easier for me to parse?  Ideally this diff would only be the things that changed.

Reply:
I think we can skip this entirely. This data is temporary.